Templates/AppSource App/.github/workflows/_BuildALGoProject.yaml

```diff
@@ -46,14 +46,10 @@ on:
         required: false
         default: ''
         type: string
-      publishThisBuildArtifacts:
-        description: Flag indicating whether this build artifacts should be published
-        type: boolean
-        default: false
-      publishArtifacts:
-        description: Flag indicating whether the artifacts should be published
-        type: boolean
-        default: false
+      artifactsRetentionDays:
```

Comment:
There's `Read settings` step in this workflow as well. Isn't it better to take the `artifactsRetentionDays` input from here?

Reply:
It is really because we don't know whether the artifacts will be long lived (CI/CD) or short lived (other workflows) in here.

Reply:
We could deduct that I guess - but this way it is explicit.